snippets/note-about-creating-tokens.adoc

```diff
@@ -0,0 +1,13 @@
+// Snippets included in the following assemblies and modules:
+//
+// * configuration/configure-api-token.adoc
+//
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+* To prevent privilege escalation, when you create a new token, your role's permissions limit you. For example, if you only have `read` permission for the Integration resource, you cannot create a token with `write` permission for that resource.
```

Comment:
```suggestion
* To prevent privilege escalation, when you create a new token, your role's permissions limit the permission you can assign to that token. For example, if you only have `read` permission for the Integration resource, you cannot create a token with `write` permission.
```